Droid/Data/CrimeLab.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Android.Content;
+using Java.Util;
+
+namespace CriminalIntentXamarin.Droid.Data
+{
+    public class CrimeLab
+    {
+        private static CrimeLab crimeLab;
+
+        private CrimeLab(Context context)
+        {
+            Crimes = new List<Crime>();
```

Comment:
Initialization can be done at declaration.

`public List<Crime> Crimes { get; } = new List<Crime>();`